Core/Core/Common/CommonUI/OptionSelection/View/MultiSelectionView.swift

```diff
@@ -86,14 +86,30 @@ public struct MultiSelectionView: View {
 
     @ViewBuilder
     private func optionCell(with item: OptionItem) -> some View {
-        InstUI.CheckboxCell(
-            title: item.title,
-            subtitle: item.subtitle,
-            isSelected: selectionBinding(for: item),
-            color: item.color,
-            accessoryView: { item.accessoryIcon?.foregroundStyle(item.color) },
-            dividerStyle: viewModel.dividerStyle(for: item)
-        )
+        if let color = item.color {
+            InstUI.CheckboxCell(
+                title: item.title,
+                subtitle: item.subtitle,
+                isSelected: selectionBinding(for: item),
+                accessoryView: {
+                    item.accessoryIcon?
+                        .foregroundStyle(color)
+                },
+                dividerStyle: viewModel.dividerStyle(for: item)
+            )
+            .tint(color)
+        } else {
+            InstUI.CheckboxCell(
+                title: item.title,
+                subtitle: item.subtitle,
+                isSelected: selectionBinding(for: item),
+                accessoryView: {
+                    item.accessoryIcon?
+                        .foregroundStyle(.tint)
+                },
+                dividerStyle: viewModel.dividerStyle(for: item)
+            )
+        }
```

Comment:
Just extract the checkboxcell into a local variable and apply tint on it if tint is available
```suggestion
        let cell = InstUI.CheckboxCell(
            title: item.title,
            subtitle: item.subtitle,
            isSelected: selectionBinding(for: item),
            accessoryView: {
                item.accessoryIcon?
                    .foregroundStyle(.tint)
            },
            dividerStyle: viewModel.dividerStyle(for: item)
        )
        
        if let color = item.color {
            cell.tint(color)
        } else {
            cell
        }
```